src/network/networkd-network.h

```diff
@@ -49,6 +49,7 @@
 typedef enum DHCPClientIdentifier {
         DHCP_CLIENT_ID_MAC,
         DHCP_CLIENT_ID_DUID,
+        DHCP_CLIENT_ID_DUID_ONLY, /* this may not good for RFC regarding DHCP (3315 and 4361). But some setup require this. See #7828. */
```

Comment:
This comment is worded awkwardly. I'd also prefer a full link rather than `#7828`, since code (and comments!) tend to outlive websites.